### DISCUSSION TITLE
FN RAT

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#392] [General]
Author: @kathypinzon 
Reviewer: @AbbyANoble 

RAT: https://www.overleaf.com/read/fpwrvykjjxzb#2cf991
Readiness: https://teambasedinquirylearning.github.io/precalculus/2024/instructor/FN.html

**Comment by AbbyANoble** [comment]
On 5, it's a little hard to tell which line the axis labels go with. Might be easier to do an axis environment in tikz to show them more clearly? (I used one in question 1 here: https://www.overleaf.com/read/frcwrfzjjpbk#414b7e)

**Reply by AbbyANoble** [reply]
Also the answer isn't a choice. Probably just a copy/paste error in (b).

**Reply by kathypinzon** [reply]
@AbbyANoble  do you know how to label the points with the letters A and B in that environment?

**Comment by AbbyANoble** [comment]
Says Module PN at the top, not FN
